docs/src/man/interval_hulls.md

```diff
@@ -0,0 +1,187 @@
+plot!(Z, 1e-3, alpha=0.2)
+
+Bapprox = ballinf_approximation(Z)
+
+plot!(Bapprox, alpha=0.1)
+plot!(Singleton(Bapprox.center), markershape=:x)
+```
+
+```@example example_ih
+Bapprox.center, Bapprox.radius
+```
+
+## Interval hull approximation
+
+If we want to have different lengths for each dimension, instead of the
+`ballinf_approximation`, we can use the approximation with a hyperrectangle through
+the `interval_hull` function.
+
+```@example example_ih
+import LazySets.Approximations.interval_hull
+
+plot(X, 1e-3, aspectratio=1)
+plot!(Y, 1e-3)
+plot!(Z, 1e-3, alpha=0.2)
+
+Happrox = interval_hull(Z)
+
+plot!(Happrox, alpha=0.1)
+plot!(Singleton(Happrox.center), markershape=:x)
+```
+
+```@example example_ih
+Happrox.center, Happrox.radius
+```
+
+!!! note
+    The `interval_hull` function is an alias for the `box_approximation` function.
+    The nomenclature for approximation functions is `*_approximation_*`. To see a
+    list of all approximation functions, either search in the docs or type
+    `names(LazySets.Approximations)`.
+
+## Symmetric interval hull
+
+Contrary to the previous approximations, the symmetric interval hull is centered
+around the origin. It is defined in the `Approximations` module as well.
+
+```@example example_ih
+import LazySets.Approximations.symmetric_interval_hull
+
+plot(X, 1e-3, aspectratio=1)
+plot!(Y, 1e-3)
+plot!(Z, 1e-3, alpha=0.2)
+
+S = symmetric_interval_hull(Z)
+plot!(S, alpha=0.2)
+plot!(Singleton(S.center), markershape=:x)
+```
+
+```@example example_ih
+S.center, S.radius
+```
+
+We can get the list of vertices using the `vertices_list` function:
+
+```@example example_ih
+vertices_list(S)
+```
+ 
+For instance, compute the support vector in the south-east direction:
+ 
+```@example example_ih
+σ([1., -1.], S)
+```
+ 
+ It is also possible to pass a sparse vector as direction, and the result is a
+ sparse vector:
+ 
+ ```@example example_ih
+σ(sparsevec([1., -1.]), S)
+```
+
+## Norm, radius and diameter
+
+In this part we illustrate some functions to obtain metric properties of sets.
+These functions apply generally to any `LazySet`. For some types, specialized
+methods are triggered automatically through multiple-dispatch.
+
+The *norm* of a convex set is the norm of the enclosing ball (of the given norm)
+of minimal volume. For instance:
+
+```@example example_ih
+import LazySets.Approximations: norm, radius, diameter
+
+norm(X), norm(Y), norm(Z)
+```
+
+The *radius* of a convex set. It is the radius of the enclosing ball
+(of the given norm) of minimal volume with the same center. In the previous example,
+
+```@example example_ih
+radius(X), radius(Y), radius(Z)
+```
+
+Finally, it is sometimes convenient to ask directly the diameter of the set,
+defined as twice the radius:
+
+```@example example_ih
+diameter(X), diameter(Y), diameter(Z)
+```
+
+In the previous examples, we have used the infinity norm (default). It is possible
+to pass the required `p`-norm as a second argument.
```

Comment:
Maybe add a short example here.

Reply:
good catch; none of them is implemented yet for p not Inf, hence i removed this sentence.